src/SuperSocket.Connection/PipeConnectionBase.cs

```diff
@@ -52,6 +52,7 @@ protected PipeConnectionBase(PipeReader inputReader, PipeWriter outputWriter, Co
             Logger = options.Logger;
             InputReader = inputReader;
             OutputWriter = outputWriter;
+            ConnectionToken = _cts.Token;
         }
 
         protected virtual Task StartTask<TPackageInfo>(IObjectPipe<TPackageInfo> packagePipe)
```

Comment:
_cts.Token can be replaced with ConnectionToken in this file. And then we will see how to improve _cts.

Reply:
<img width="1385" alt="image" src="https://github.com/kerryjiang/SuperSocket/assets/39239954/bf3ac1c5-5f7f-4e49-b6cd-5844cac4a4a0">
Is that OK?

Reply:
<img width="1134" alt="image" src="https://github.com/kerryjiang/SuperSocket/assets/39239954/fdbcb744-8a83-46bc-8d44-b0b922e0bfa0">
That should be even better.